src/schema/schema.ts

```diff
@@ -274,6 +274,16 @@ export function createZimbraSchema(
 								_attrs: mapValues(variables.attrs, coerceBooleanToString)
 							}
 						}
+					}),
+				setRecoveryAccount: (
+					_,
+					{ channel, op, recoveryAccount, recoveryAccountVerificationCode }
```

Comment:
Similar to above, this can be shortened to `variables as SetRecoveryAccountOptions` and all variables can be passed on to `client.setRecoveryAccount`.